src/classes/User.js

```diff
@@ -0,0 +1,41 @@
+class User {
+  constructor(userData = {}) {
+    this.id = userData.id;
+    this.name = userData.name;
+    this.pantry = userData.pantry;
+    this.recipesToCook = [];
+  }
+
+  addRecipesToCook(recipe) {
+    this.recipesToCook.push(recipe);
+  }
+  
+  removeRecipesToCook(recipe) {
+    let recipeToRemove = this.recipesToCook.indexOf(recipe);
+    this.recipesToCook.splice(recipeToRemove, 1);
+    return this.recipesToCook;
+  }
+  filterRecipeByTag(tag) {
```

Comment:
since this method returns an array of recipes instead of one recipe, I think it would be good to rename it from the singular to the plural `filterRecipe**s**ByTag`

Reply:
This has been adjusted to the plural.